tests/meta/workflow/test_analysis.py

```diff
@@ -566,3 +578,57 @@ def test_link_fastq_files_for_sample(
 
         # THEN broadcast linking of files
         assert "Linking: " in caplog.text
+
+
+def test_are_all_samples_control(analysis_store: Store, case_id: str) -> None:
+    """Tests that are_all_samples_control returns True if all samples in a case are controls."""
+
+    # GIVEN a case with all samples being positive controls
+    case: Case = analysis_store.get_case_by_internal_id(case_id)
+    for sample in case.samples:
+        sample.control = ControlOptions.POSITIVE
+
+    # WHEN checking if all samples are controls
+    # THEN the result should be True
+    assert are_all_samples_control(case) == True
```

Comment:
```suggestion
    assert are_all_samples_control(case)
```
Since `are_all_samples_control` already returns a bool there is no need to do a comparison here.